vue-app/src/views/Profile.vue

```diff
@@ -69,9 +69,38 @@
           </balance-item>
         </div>
       </div>
-      <div class="projects-section">
+      <div v-if="projects.length > 0" class="projects-section">
         <h2>Projects</h2>
-        <!-- <div class="project-item" v-for=" eacah project user owns " /> -->
+        <div class="project-list">
+          <div
+            class="project-item"
+            v-for="{
+              id,
+              name,
+              thumbnailImageUrl,
+              isHidden,
+              isLocked,
+            } of projects"
+            :key="id"
+          >
+            <img
+              :src="thumbnailImageUrl"
+              :alt="alt + ' thumbnail'"
+              class="project-thumbnail"
+            />
+            <div class="project-details">
+              <div class="project-name">
+                {{ name }}
+                <span v-if="isLocked">🔒</span>
+              </div>
+              <div v-if="isHidden" class="project-hidden">Under review</div>
+            </div>
+            <button class="btn-secondary" @click="navigateToProject(id)">
+              {{ isLocked ? 'Preview' : 'View' }}
+            </button>
+          </div>
+          <loader v-if="isLoading" />
```

Comment:
I don't think this loader ever displays because while this data is loading the `v-if="projects.length > 0"` in the parent div always evaluates to false.

Reply:
This makes the interaction a bit awkward as there's always a delay before the info appears:

https://user-images.githubusercontent.com/8097623/131417390-23571561-7a4c-439c-94f0-fa3625df5f9e.mp4

Should we have the loading state appear for all users & display "No projects" or something if the user hasn't submitted any? That would at least avoid this jerky UX.

Reply:
Maybe we always display the "Projects" header, then have an initial loading spinner underneath. If we resolve the query & the user has no projects, we can say "You have submitted any projects" in text underneath the "Projects" header.